packages/rocketchat-livechat/.app/client/views/message.html

```diff
@@ -10,7 +10,13 @@
 				{{/if}}
 			</span>
 			<div class="body" dir="auto">
-				{{{body}}}
+				{{#if hasAttachments}}
```

Comment:
I think this `{{#if ... }}` can be removed.

instead it should always shows `{{{body}}}` and then have the `{{#each attachments}}`.. this will allow it showing replies from rocket.chat (somewhat).

Reply:
Fixed, but we will need to thing about improving `markdown` on Livechat, I guess.. =)